tests/phpunit/item/ItemTest.php

```diff
@@ -61,6 +65,16 @@ public function testItemBlockRegistered() : void{
 		}
 	}
 
+	/**
+	 * Tests that when all enchantments are removed from an item, the "ench" tag is removed as well
+	 */
+	public function testEnchantmentRemoval() : void{
+		$item = ItemFactory::get(Item::DIAMOND_SWORD);
+		$item->addEnchantment(new EnchantmentInstance(Enchantment::getEnchantment(Enchantment::SHARPNESS)));
+		$item->removeEnchantment(Enchantment::SHARPNESS);
+		self::assertFalse($item->getNamedTag()->hasTag(Item::TAG_ENCH, ListTag::class));
```

Comment:
```suggestion
		self::assertNull($item->getNamedTag()->getTag(Item::TAG_ENCH));
```
PM4's NBT doesn't have `hasTag()`, and this version is better suited to both versions.